pkg/errutils/errutils.go

```diff
@@ -28,6 +28,7 @@ const (
 	ErrRequireArg             = "command requires an input file argument. Run '%s' for more information about a command"
 	ErrInvalidVersionString   = "invalid version %s. Expected %s"
 	ErrInvalidNumJobs         = "invalid number of job slots specified for parallel execution. Expected: j>0"
+	ErrMissingRequiredArg     = "missing required argument. Use --context-set option"
```

Comment:
"setup command is missing mandatory option '--context-set'"

Reply:
[Addressed review comment](https://github.com/Open-CMSIS-Pack/cbuild/pull/259/commits/b19e3fcf826e9caec827f69bd54d43120bb19e74)